src/utils/convert.js

```diff
@@ -224,6 +280,13 @@ const renderAuth = (definition) => {
   }
 };
 
+// write authentiction.js
```

Comment:
Minor thing, but there's a typo in this comment `authentication.js` is what it should be.

Reply:
Fixed in 9212231.